docs/user-docs/latest/plugins/introduction-to-plugins.md

```diff
@@ -6,12 +6,16 @@ canonicalUrl: https://docs.strapi.io/user-docs/latest/plugins/introduction-to-pl
 
 # Introduction to plugins
 
-Strapi is built around plugins of different kinds. There are the core plugins which are essential for your Strapi application to function, and therefore cannot be deactivated. But there are also other plugins, that can either be installed by default or not, to add more options and possibilities to your Strapi application.
+Strapi is built around different types of plugins. Core plugins are essential for your Strapi application to function, and therefore cannot be deactivated. Pre-installed plugins, such as Internationalization, are installed in a default Strapi application. Additional plugins that extend and customize Strapi applications are available in the [Marketplace](../plugins/installing-plugins-via-marketplace.md).
 
 ::: note
-Core plugins which include the Content Manager and the Content-type Builder are documented in their own sections of the user guide as they are quite specific (see [Introduction to the Content Manager](../content-manager/introduction-to-content-manager.md) and [Introduction to the Content-type Builder](../content-types-builder/introduction-to-content-types-builder.md)). This Plugins section focuses on how to manage plugins in general and provides documentation for the others, non-core plugins.
+Core plugins which include the Content Manager and the Content-type Builder are documented in their own sections of the user guide as they are quite specific (see [Introduction to the Content Manager](../content-manager/introduction-to-content-manager.md) and [Introduction to the Content-type Builder](../content-types-builder/introduction-to-content-types-builder.md)). This Plugins section focuses on how to manage plugins in general and provides documentation for the other, non-core plugins.
 :::
 
-Plugins are managed, for anything related to installation and deletion, from ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_ and ![Plugins icon](../assets/icons/plugins.svg) _Plugins_, both available in the main navigation of the admin panel. Once installed, your plugins are added to the main navigation of the admin panel, from where they are directly accessible.
+Plugins installation and deletion are managed from the CLI (see [Developer Documentation](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-install)). In the main navigation of the admin panel:
+
+- Pre-installed plugins and are managed through ![Settings icon](../assets/icons/settings.svg) _Settings_.
+- Additional plugins are discoverable in the ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_.
+- All installed plugins are listed under ![Plugins icon](../assets/icons/plugins.svg) _Plugins_.
```

Comment:
Not sure about this part because it mixes mentioning "everything is handled from the CLI" but then going straight to a bullet list about the 3 parts of the admin panel that are about plugins 🤔 
I like the bullet list but I'd change `In the main navigation of the admin panel` I think, it sounds weird with the following sentences. And I think we need to figure out a better way to explain that from now on no plugin can be directly installed from the Marketplace. Let's discuss that together 😄